### PR TITLE
Finished html-tables exercise. Lambert Nguyen

### PR DESCRIPTION
**Pull request by Ln1097**
Screenshot (215)

### CODE REVIEW

File: html-tables.tar/html-tables/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  <head>
+  </head>
+  <body>
+    <h1>Student Grade Table</h1>
+<table>
+  <tr>
+    <th>Student Name</th>
+    <th>Course</th>
+    <th>Grade</th>
+  </tr>
```
Comment (kurtyazdizadeh) [inline html-tables.tar/html-tables/index.html:12]: there should be a `<thead>` element wrapping this `<tr>` element

File: html-tables.tar/html-tables/index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    <td>HTML</td>
+    <td>9001</td>
+  </tr>
+   <tr>
+    <td>Linus Torvalds</td>
+    <td>Git</td>
+    <td>90</td>
+  </tr>
+</table>
+</tr>
```
Comment (kurtyazdizadeh) [inline html-tables.tar/html-tables/index.html:39]: there should be a`<tbody>` element wrapping this `<tr>` element.

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
Very close! Just needs the missing `thead` and `tbody` elements to be semantically correct.

**Comment by Ln1097** [comment]
fixed

**Review by kurtyazdizadeh — APPROVED** [review]
Approved 👍

**Pull request merged**